### DISCUSSION TITLE
Верстак

### DISCUSSION THREAD

**Discussion by glitchcore** [b4ck5p4c3/0x08.in#359] [General]
Предлагается установить верстак в HW-зоне, у окна, напротив токарника.

Два варианта изготовления:
1. Самостоятельно изготовить из фанеры и бруса (модель есть у @sntxerror)
2. Заказать (инфа у @adjofun)

Верстак необходимо закрепить к стене. Предлагаю сделать это с помощью цанговых анкеров и шпилек, преварительно просверлив отверстия под шпильки в гипсокартоне. Это позволит в будущем безболезненно демонтировать верстак и обеспечит надежное крепление его непосредственно к бетонной стене.

**Comment by sntxerror** [comment]
Я все еще сомневаюсь про анкеры, и голосую за вырезы под брус в ГКЛ и за крепление бруса к стене вплотную. В случае с анкерами будет всеравно немало отверстий, плюс я боюсь что они для другого рода нагрузки подходят, для растяжения.

**Reply by glitchcore** [reply]
Заделать 8 отверстий намного проще, чем восстанавливать вырезы в ГК.

С вырезами связано два момента:

1. Мне нужен человек, которй возьмет на себя восстановление стены.
2. Резать стену сопряжено с большим хаосом чем сверлить в ней отверстия.

Не разделяю твои опасения относительно характера нагрузок. Даже для 8.8 болта M12 заявлено усилие отрыва 6 тонн.

**Comment by ghost** [comment]
Вот коммерческое предложение от конторы.
Решение по заказу надо принять до 24.02.2021 включительно.
В стоимость не включена доставка из Казани. 

![IMG-20210219-WA0006](https://user-images.githubusercontent.com/73026364/108688813-608de700-7509-11eb-94c0-31cffa645058.jpg)
![IMG-20210219-WA0005](https://user-images.githubusercontent.com/73026364/108688816-608de700-7509-11eb-8667-842b3a9fa9e3.jpg)
![IMG-20210219-WA0004](https://user-images.githubusercontent.com/73026364/108688818-61267d80-7509-11eb-8bc1-dd13f7c56abf.jpg)
![IMG-20210219-WA0003](https://user-images.githubusercontent.com/73026364/108688820-61267d80-7509-11eb-86db-8eb6936b2010.jpg)
![IMG-20210219-WA0002](https://user-images.githubusercontent.com/73026364/108688821-61bf1400-7509-11eb-8de7-6cc077dcca8d.jpg)
![Screenshot_20210219_151221](https://user-images.githubusercontent.com/73026364/108688959-8dda9500-7509-11eb-8cf2-9d4f2f392a70.jpg)

**Comment by rusdacent** [comment]
Вот как в SketchUp делают подготовку для распила

Расширение (плагин) SketchUp для проектирования мебели программистом

> Решение достаточно простое — создаю некоторый API, который позволяет создавать 3D объекты листов ЛДСП с нужными параметрами. Из этих объектов компонуется готовая модель а также в отдельный файл выгружаются размеры листов для передачи их в центр распила. 

https://habr.com/ru/post/442190/

**Comment by z4fd** [comment]
# Концептуально у нас есть два пути:

### Купить готовый верстак, у этой затеи есть проблемы:
- Хреново впишется, не подходит из за выступа на полу
- Дорогой + еще доставка. 40к + доставка
- Доставка и установка может быть проблемой (втащить, установить, собрать и т.д. )

### Сделать свой верстак из фанеры, главный плюс этой затеи:
- Собирать свой верстак будет весело :)




### Сейчас остановились на самодельном верстаке, для продолжение этой темы нужно:
- Закончить проектирование верстака
- Оценить расход материалов и решить проблем с креплением в стены
- Обдумать крепление в пол

### Нужно определиться с крепление верстака к стене:
#### Крепление на анкера через гипрок к стене 
- Есть опасения что они будет упруго деформироваться
- Нужно проверить отсутствие пеноблоков в стене
  - Предполагается решать её экспериментальным просверлив пару дырок. Результат добавить суда
#### Демонтаж гипрока
 - Грязный демонтаж
 - Последующая отделка стены
 - В случае освобождения помещения нужно будет восстановить стену (Этот пункт бьется только, если есть есть человек готовый взять на себя отверженность сделать это)


#### Заметки:
6 см  -  расстояние от стены, до наружной поверхности гипрока

з.ы.: Возможны апдейты, не является итоговым решение. Комментарии и пожелания приветствуются :)

**Comment by z4fd** [comment]
## Модели верстака в stl и skp
![photo_2021-02-18_23-36-57](https://user-images.githubusercontent.com/32653368/108912630-f45ed080-7639-11eb-97c6-c0df7902786e.jpg)
![photo_2021-02-24_00-49-05](https://user-images.githubusercontent.com/32653368/108912807-2d974080-763a-11eb-93ad-6d124140163b.jpg)
![photo_2021-02-24_00-49-01](https://user-images.githubusercontent.com/32653368/108912808-2ec86d80-763a-11eb-9fa1-5f9fdc0993ad.jpg)

Можно открыть через https://app.sketchup.com/app
[HW Worbench.stl.zip](https://github.com/b4ck5p4c3/0x08.in/files/6031993/HW.Worbench.stl.zip)
